.github/workflows/configure.yml

```diff
@@ -13,6 +13,11 @@ jobs:
             compiler: gcc
             configure-args: --warn
 
+          - name: Ubuntu 16.04 GCC
+            os: ubuntu-16.04
```

Comment:
This uses oldest ubuntu image.. 
Can we instead do `packages: gcc-5` and `compiler: gcc-5`?

Reply:
An explicit goal of this PR is to test on the oldest supported operating system.